docs/about/contributing.md

```diff
@@ -28,12 +28,12 @@ Once you have forked the repository, you can now clone your forked repository to
 To clone your forked repository, first make sure you have installed [Git](https://git-scm.com/downloads), the version control system used by GitHub. Then open a Terminal and type the following commands:
 
 ```bash
-mkdir zio
-cd zio
-git clone git@github.com:your-user-name/zio.git .
+mkdir zio-prelude
```

Comment:
AFAIU it, this should stay `zio`. The point is to have a dedicated directory for all repositories under the https://github.com/zio organization.